ntbs-integration-tests/NotificationPages/OverviewPageTests.cs

```diff
@@ -36,5 +38,44 @@ public async Task GetOverviewPage_ReturnsCorrectStatusCode_DependentOnId(int id,
                 Assert.Equal(BuildRoute(Routes.Patient, id), GetRedirectLocation(response));
             }
         }
+
+        [Fact]
+        public async Task Get_ReturnsOverviewPage_ForUserWithPermission()
+        {
+            // Arrange
+            var idToServiceCodeMap = new Dictionary<int, string>
+            {
+                { Utilities.NOTIFIED_ID, Utilities.PERMITTED_SERVICE_CODE }
+            };
+            var client = factory.WithNhsUserBuilder(idToServiceCodeMap).WithoutRedirects();
+
+            //Act
+            var response = await client.GetAsync($"{GetPageRouteForId(Utilities.NOTIFIED_ID)}");
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            var document = await GetDocumentAsync(response);
+            // There are 8 sections on the overview page
+            Assert.Equal(8, document.GetElementsByClassName("notification-overview-type-and-edit-container").Length);

```

Comment:
Seems like a very fragile test condition..
I wouldn't mind if this just checked the label or a value, or a section title, and we'll have to accept that content is always prone to change, but number specifically seems like the most likely one to change repeatadly.

Reply:
Do you think
`document.GetElementsByClassName("notification-overview-type-and-edit-container").Length > 0` would be fine? Then we just make sure we have at least some content with this class type

Reply:
yeah, seems fine that way :)